code/obj/item/clothing/gimmick.dm

```diff
@@ -419,6 +419,8 @@
 // it was called the blessed clown for the like half week it existed before
 
 /obj/item/clothing/mask/clown_hat/blue
+	bald_icon_state = "blueclownbald"
```

Comment:
change this & sprite name to blessedclownbald, having two different names is bad